common/screenshotter.go

```diff
@@ -300,7 +300,7 @@ func (s *screenshotter) screenshotElement(h *ElementHandle, opts *ElementHandleS
 	return buf, nil
 }
 
-//nolint:funlen,cyclop
+//nolint:funlen,cyclop,gocognit
```

Comment:
```suggestion
//nolint:linter
```

😄

Reply:
I'll leave it as that for now 😄

Reply:
But, I have a strong opinion :trollface: :trollface: :trollface: